nntrainer/src/neuralnet.cpp

```diff
@@ -1035,6 +1055,7 @@ std::shared_ptr<Layer> NeuralNetwork::_make_act_layer(ActiType act,
     std::shared_ptr<ActivationLayer> act_layer =
       std::make_shared<ActivationLayer>();
 
+    act_layer->setName(prev->getName() + "Activation");
     act_layer->setActivation(act);
```

Comment:
No related with this PR directly, I think when you have to make setLayer property after get the layer with this getLayer,  we have to extra concern about this Activation. if we change the activation, then we have to change  this activation layer as well.

Reply:
Yes, thats correct. May be create activation layer once the model is final and fixed.